vars/pluginUtils.groovy

```diff
@@ -0,0 +1,88 @@
+def runPluginTests(plugins, envName, markerName, slackChannel, slackUrl) {
```

Comment:
can we call `plugins` `pluginsConfig`... I was confused at first and thought it'd just be a list of plugin names :)